### ISSUE TITLE
Build error under Debian bullseye

### ISSUE THREAD

**Issue by rdebath** [ClassiCube/MCGalaxy#764]
https://github.com/UnknownShadow200/MCGalaxy/blob/61a332181d35d5f0aa9d4aac1f108fd6d9df7820/GUI/Popups/PortTools.Designer.cs#LL106C64-L106C76

```
XBuild Engine Version 14.0
Mono, Version 6.8.0.105
Copyright (C) 2005-2013 Various Mono authors

Build started 06/10/2023 12:55:37.
...
Popups/PortTools.Designer.cs(106,64): error CS1061: Type `MCGalaxy.Gui.Popups.PortTools' does not contain a definition for `LblInfoClick' and no extension method `LblInfoClick' of type `MCGalaxy.Gui.Popups.PortTools' could be found. Are you missing an assembly reference?
                                Popups/PortTools.cs(25,26): (Location of the symbol related to previous error)
                        Task "Csc" execution -- FAILED
...

```

**Comment by UnknownShadow200** [comment]
Will be fixed when I get around to making the next actual MCGalaxy commit (I didn't want to make an early separate commit just to fix one line of code)

**Comment by UnknownShadow200** [comment]
Fixed as part of 28843b10c466c532282b9409ee25c8a9348c5580
